src/Raven.Studio/typescript/models/database/index/indexDefinition.ts

```diff
@@ -196,6 +208,7 @@ class indexDefinition {
             Configuration: this.configurationToDto(),
             Fields: this.fieldToDto(),
             OutputReduceToCollection: this.outputReduceToCollection() ? this.reduceToCollectionName() : null,
+            PatternForOutputReduceToCollectionReferences: this.patternForOutput() ? this.patternForOutputReduceToCollectionReferences() : null,
```

Comment:
should we check for this.outputReduceToCollection() as well?

Reply:
yes, done